src/components/PollList/pollList.js

```diff
@@ -0,0 +1,104 @@
+import "./pollList.css";
+import { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import { useDispatch, useSelector } from "react-redux";
+import { Button, Form, Container, Navbar } from "react-bootstrap";
+import { Trash, PencilSquare } from "react-bootstrap-icons";
+import pollList from "../../redux/pollList/actions/pollList";
+import { removeUserData } from "../../redux/login/actions/login";
+
+const PollList = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const [pollIDs, setPollIDs] = useState([]);
+  const pollQuestion = useSelector((state) => state.pollList.pollList);
+  const userDetailsFromLocalStorage = JSON.parse(localStorage.getItem("user"));
```

Comment:
userDetailsFromLocalStorage this data should not come from local storage, it should come from Store. @PuniaMohit